ts/src/header-validator/validate-json.ts

```diff
@@ -1132,12 +1169,34 @@ function source(j: Json, ctx: SourceContext): Maybe<Source> {
           'destination_limit_priority',
           withDefault(int64, 0n)
         ),
+        attributionScopeLimit: field(
+          'attribution_scope_limit',
+          withDefault(positiveUint32, null)
+        ),
+        attributionScopes: field(
```

Comment:
Wdyt about passing the parsed `attributionScopeLimit` to validate `attribution_scopes` and `max_event_states`?

Reply:
Done. thanks, it looks much cleaner